dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/organisationunit/DefaultOrganisationUnitService.java

```diff
@@ -335,6 +341,8 @@ public OrganisationUnit getOrganisationUnitByCode( String code )
         params.setMaxLevels( levels );
         params.setFetchChildren( true );
 
+        params.setOrderBy( systemSettingManager.getSystemSetting( SettingKey.ANALYSIS_DISPLAY_PROPERTY, "name" ).toString().toLowerCase() );

```

Comment:
Should not set the default value here directly. The system setting manager will return the default value as specified by SettingKey.defaultValue if no value is set. Keeps the default value centralized.